proto/config.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto2";
+package proto;
+
+option go_package = "github.com/googleinterns/aso_sxs_viewer/proto";
+
+message ViewerConfig {
+    optional int32 browser_window_count = 1;
+    optional CSSSelector css_selector = 2; //search bar selector
+    optional string url = 3;
+    optional bool use_cookies = 4;
+    optional string user_data_dir_prefix = 5;
+    repeated BrowserConfig window_overrides = 6;
+    enum WindowPosition {
+        BOTTOM = 0; //default
+        TOP = 1; 
+    }
+
+    optional WindowPosition input_window_position = 7;
+    // RootWindowConfig is used only for grouping windows in X based systems.
+    optional RootWindowConfig root_window_config = 15;
+}
+message BrowserConfig {
+    optional CSSSelector css_selector = 1;
```

Comment:
reverse the order here (url first, selector later). Also do this in the parent proto